aries_cloudagent/ledger/indy.py

```diff
@@ -886,7 +886,12 @@ async def get_endpoint_for_did(
         return address
```

Comment:
You'll need to make these same changes in the indy_vdr.py version of the ledger

(this is enabled when you use `--wallet-type askar`)